src/components/groups/ConfirmAutoFill.tsx

```diff
@@ -0,0 +1,44 @@
+import React from "react";
+
+interface createProps {
+  handleAutoFill: () => void;
+  handleClose: () => void;
+}
+
+const ConfrimationCard: React.FC<createProps> = ({
+  handleAutoFill,
+  handleClose,
+}) => {
+  const handleConfirm = () => {
+    handleAutoFill();
+  };
+
+  return (
+    <div className="flex flex-col mb-4  w-[50vw] md:w-[40vw] lg:w-[30vw]">
+      <div className="info mx-auto items-center justify-content text-start">
+        <h1 className="font-bold p-2 justify-center">Are you sure?</h1>
+        <p className="text-light item-align justify-center">
```

Comment:
`justify-center` do the same as the above comment and in Tailwind, there is no class called `item-align`.  so remove both classes